src/main/java/mServer/crawler/sender/newsearch/ZDFSearchTask.java

```diff
@@ -17,7 +17,6 @@
     private static final String PROPERTY_SEARCHPARAM_Q = "q";
     private static final String SEARCH_ALL = "*";
     private static final String PROPERTY_TYPES = "types";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "PROPERTY_TYPES" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)